ProQuest.js

```diff
@@ -120,79 +120,96 @@ function fetchEmbeddedPdf(url, item, callback) {
 }
 
 function getSearchResults(doc, checkOnly) {
-	var tabs = doc.getElementsByClassName('tabContent');
 	var root;
-	if (tabs.length) {
-		for (var i = 0; i < tabs.length; i++) {
-			if (tabs[i].offsetHeight) {
-				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
-				root = tabs[i].getElementsByClassName('resultListContainer')[0];
+	var tabs = doc.getElementsByClassName('tabContent');
+	for (var i = 0; i < tabs.length; i++) {
+		if (tabs[i].offsetHeight) {
+			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
+			root = tabs[i].getElementsByClassName('resultListContainer')[0];
+			break;
+		}
+	}
+	if (!root) {
+		var elements = doc.getElementsByClassName('resultListContainer');
+		for (var i=0; i<elements.length; i++) {
+			if (elements[i] && elements[i].offsetHeight>0) {
```

Comment:
There are pages with two `resultListContainer`s corresponding to the two tabs in a search (one is ebrary):
![proquest](https://cloud.githubusercontent.com/assets/5199995/9450377/932611c4-4aa8-11e5-9618-83afd11897c7.png)
However, I don't know if we still need to check for some element `root` which is not used any further.